### PR TITLE
Simulator observers endpoint

### PR DESCRIPTION
**Pull request by miiu96**
- Added an extra endpoint `simulator/observers` that will return information about observers that are behind chain simulator

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,36 @@ This endpoint returns the initial wallets (address and private key hex encoded).
 }
 ```
 
+### `GET /simulator/observers`
+
+This endpoint returns the initial wallets (address and private key hex encoded).
```
Comment (iulianpascalau) [inline README.md:112]: wrong description?

File: pkg/dtos/observers.go [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package dtos
+
+type ObserverInfo struct {
```
Comment (iulianpascalau) [inline pkg/dtos/observers.go:3]: missing comment

File: pkg/facade/simulatorFacade.go [hunk 3]
```diff
@@ -93,6 +95,30 @@ func (sf *simulatorFacade) ForceUpdateValidatorStatistics() error {
...
 
+// GetObserversInfo will return information about the observers
+func (sf *simulatorFacade) GetObserversInfo() (map[uint32]*dtoc.ObserverInfo, error) {
+	restApiInterface := sf.simulator.GetRestAPIInterfaces()
+
+	response := make(map[uint32]*dtoc.ObserverInfo)
+	for shardID, apiInterface := range restApiInterface {
+		split := strings.Split(apiInterface, ":")
+		if len(split) != 2 {
+			return nil, fmt.Errorf("cannot extrac port for shard ID=%d", shardID)
```
Comment (iulianpascalau) [inline pkg/facade/simulatorFacade.go:106]: extract *

File: README.md (hunk 1 above)
Comment (miiu96) [reply README.md:112]: changed

File: pkg/dtos/observers.go (hunk 2 above)
Comment (miiu96) [reply pkg/dtos/observers.go:3]: added

File: pkg/facade/simulatorFacade.go (hunk 3 above)
Comment (miiu96) [reply pkg/facade/simulatorFacade.go]: done

**Review by iulianpascalau — APPROVED** [review]

**Review by ssd04 — APPROVED** [review]

**Pull request merged**